frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationConstants.ts

```diff
@@ -76,11 +76,31 @@ export const FREQUENCIES = [
 		value: `${SECONDS_PER_DAY * 3}`,
 		id: '3d',
 	},
+	{
+		name: '5 days',
```

Comment:
Tagging @mayberryzane - are the alerts on the backend only looking back at a max of 1 hour currently? Think its okay for the graphing interval, but wondering if we should have a copy or manipulate the list for alerts?

Reply:
during alert evaluation we aggregate data +/- 2 hours from the current date, then use the alert's threshold window to filter. I think there could be some issues for longer time ranges after a change is made to the alerts setting until there's `thresholdWindow` amount of data saved

Reply:
:+1: will pull this out into a new dashboard frequencies array, tho it sounds like we may already have issues with this with the larger ranges?